modules/lldp/control.c

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include "gr_lldp.h"
+
+#include <gr_api.h>
+#include <gr_control.h>
+#include <gr_graph.h>
+#include <gr_iface.h>
+#include <gr_log.h>
+#include <gr_worker.h>
+
+#include <event2/event.h>
+#include <rte_common.h>
+#include <rte_graph_worker.h>
+
+int lldp_output_request_solicit(const struct iface *iface);
+
+extern struct gr_lldp_node_data lldp_ctx;
+extern struct gr_lldp_neigh neighbors[];
```

Comment:
I didn't find this declared in any header. Maybe it would be best to keep the symbol private to `sink.c` and expose it via functions.

Reply:
ack.